content/applications/finance/accounting/taxes/avatax.rst

```diff
@@ -166,28 +166,49 @@ Next, ensure that the Odoo *AvaTax* module is installed. To do so, navigate to t
+       in Odoo.
    * - :guilabel:`Avatax for SO`
      - `account_avatax_sale`
      - Includes the information needed for tax calculation on sales orders in Odoo.
-   * - :guilabel:`Avatax for Subscriptions`
-     - `account_avatax_sale_subscription`
-     - This module includes the features required for tax calculation on subscriptions in Odoo.
-   * - :guilabel:`Account Avatax - Ecommerce`
-     - `website_sale_account_avatax`
-     - Includes tax calculation features for the checkout process on Odoo eCommerce.
-   * - :guilabel:`Account AvaTax - Ecommerce - Delivery`
-     - `website_sale_delivery_avatax`
-     - Includes tax calculation features for the delivery process on Odoo eCommerce.
-
-Click the :guilabel:`Install` button on the module labeled :guilabel:`Avatax`: `account_avatax`.
+   * - :guilabel:`Avatax for Inventory`
+     - `account_avatax_stock`
+     - Includes tax calculation in Odoo Inventory.
+   * - :guilabel:`Amazon/Avatax Bridge`
+     - `sale_amazon_avatax`
+     - Includes tax calculation features between the *Amazon Connector* and Odoo.
+   * - :guilabel:`Avatax Brazil`
+     - `l10n_br_avatax`
+     - Includes information for tax calculation in the Brazil localization.
+   * - :guilabel:`Avatax Brazil for Services`
+     - `l10n_br_avatax_services`
+     - This module includes the required features for tax calculation for services in the Brazil
+       localization.
+   * - :guilabel:`Avatax Brazil Sale for Services`
+     - `l10n_br_edi_sale_services`
+     - This module includes the required features for tax calculation for the sale of services in
+       the Brazil localization. This includes electronic data interchange (EDI).
+   * - :guilabel:`Test SOs for the Brazilian AvaTax`
+     - `l10n_br_test_avatax_sale`
+     - This module includes the required features for test sales orders in the Brazil localization.
+
+Click the :guilabel:`Install` button on the module labeled, :guilabel:`Avatax`: `account_avatax`.
 Doing so installs the following modules:
 
 - :guilabel:`Avatax`: `account_avatax`
 - :guilabel:`Avatax for SO`: `account_avatax_sale`
-- :guilabel:`Account Avatax - Ecommerce`: `website_sale_account_avatax`
+- :guilabel:`Avatax for Inventory`: `account_avatax_stock`
 
-Should *AvaTax* be needed for Odoo *Subscriptions*, or for delivery tax in Odoo *eCommerce*, then
-install those modules individually by clicking on :guilabel:`Install`.
+Should *AvaTax* be needed for geo-localization, or with the *Amazon Connector*, then install those
+modules individually by clicking on :guilabel:`Install` on :guilabel:`Avatax for geo localization`
+and :guilabel:`Amazon/Avatax Bridge`, respectively.
+
+.. tip::
+   To install *AvaTax* for Brazil, click :guilabel:`Install` on the :guilabel:`AvaTax Brazil`
+   module. This action installs :guilabel:`AvaTax Brazil` and :guilabel:`AvaTax Brazil for
+   Services`. Install the additional Brazil *Avatax* modules as needed.
```

Comment:
```suggestion
.. tip::
   To install the *AvaTax* localization for Brazil, click :guilabel:`Install` on the :guilabel:`AvaTax Brazil`
   module. This action installs :guilabel:`AvaTax Brazil` and :guilabel:`AvaTax Brazil for
   Services`. Install the additional Brazil *Avatax* modules as needed.
   
  .. seealso::
    Reference the :doc:`Brazilian localization documentation <../../fiscal_localizations/brazil>` for more specifics on Brazilian localization. 
```

Reply:
Thanks @tiku-odoo, and curious on your thoughts here @StraubCreative!

Here is the PR @vbe-odoo and I are working on, that details the l10n_br electronic invoicing & tax computation flow https://github.com/odoo/documentation/pull/8784/. 

Just my two cents– I think it is a good idea to list __all__ of the AvaTax modules that are available, even Brazil, in this main AvaTax document, but leave the local-specific install/config instructions to the fiscal localization docs, even though Brazil is the only local besides USA & Canada that Odoo supports for AvaTax.

Here is what I am suggesting, instead of the tip admonition that details Brazil AvaTax modules:
```suggestion
.. seealso::
   For localization specific *AvaTax* instructions, view the following :doc:`fiscal localization
   <../../fiscal_localizations>` documentation:

   - :doc:`../../fiscal_localizations/brazil`
```
Granted, this can be sorted out in another [IMP] PR on either this avatax.rst doc, or the brazil.rst fiscal localization doc so that we can get this important information live for 17.0+.